src/main/java/org/reactome/release/qa/check/HumanReactionsWithNonHumanComplexesWithHumanComponentsCheck.java

```diff
@@ -0,0 +1,100 @@
+package org.reactome.release.qa.check;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.reactome.release.qa.annotations.SliceQATest;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QACheckerHelper;
+import org.reactome.release.qa.common.QAReport;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * QA check that finds non-human Complexes that contain human Components, that are members of human, non-manually inferred ReactionlikeEvents.
+ */
+
+@SliceQATest
+public class HumanReactionsWithNonHumanComplexesWithHumanComponentsCheck extends AbstractQACheck {
+    @Override
+    public QAReport executeQACheck() throws Exception {
+        QAReport report = new QAReport();
+        QACheckerHelper.setHumanSpeciesInst(dba);
+        QACheckerHelper.setSkipList(null);
```

Comment:
I'm just wondering if this line, in this and other classes, is needed since the skip list will be an empty list by default?

Reply:
Good catch, I'll remove that line